src/App.js

```diff
@@ -1,9 +1,20 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom"
 import AddTooForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 
 function App() {
+  return (<BrowserRouter>
+    <Routes>
+      <Route path="/" element={<MainList />} />
```

Comment:
This is the only way it works. I found this at Router V6 tutorial.

Try to put anything inside `<Route>` shows nothing. Seems [documentation](https://reactrouter.com/en/main/route/route) is not ready yet:

![image](https://github.com/AnnaLuky/react-todo/assets/127468552/3ed81a25-5a01-4fe3-8ad6-7555bade3520)